cluster/cluster-up-cpu

```diff
@@ -9,12 +9,17 @@ echo "Using Project:      ${PROJECT}"
 echo "Using Zone:         ${ZONE}"
 echo "Using Cluster Name: ${CLUSTER_NAME}"
 echo "Using K8S Version:  ${K8S_VERSION}"
+echo "Number of Nodes:    ${NUM_K8S_NODES}"
+
+export PARALLELISM="$((3 * ${NUM_K8S_NODES}))"
```

Comment:
Eep.  Having parallelism defined as an env var is great when N is integers greater than one -- how's your bash math with floating points?  (For GPU, parallelism is usually 98% or 95% of number of nodes)

Reply:
I think Parallelism is an int in the Kubernetes job api